src/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2017-2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+# minimum required CMAKE version
+CMAKE_MINIMUM_REQUIRED(VERSION 3.7 FATAL_ERROR)
```

Comment:
You have a minimum version of 3.7 here, but a version check for `3.1.0` in the `setup.py` script.  Not sure what is going on there?

Reply:
These were copied from examples. I guess I should just use 3.7.

Reply:
I've changed the check in `setup.py` to match the `CMakeLists.txt` version.